pkg/innerring/processors/netmap/process_epoch.go

```diff
@@ -48,6 +48,7 @@ func (np *Processor) processNewEpoch(epoch uint64) {
 	np.handleNewAudit(audit.NewAuditStartEvent(epoch))
 	np.handleAuditSettlements(settlement.NewAuditEvent(epoch))
 	np.handleAlphabetSync(governance.NewSyncEvent())
+	np.handleNotaryDeposit(nil)
```

Comment:
I suggest do one of:

- make `NotaryDepositHandler` of type `func()`
- pass causal event to it (new epoch event?).
This will avoid potential panics.

Reply:
new epoch event seems ok.